flow2ml/README.md

```diff
@@ -14,6 +14,7 @@ Flow2Ml
         ├──Data_Loader.py
         ├──Filters.py  
         ├──Data_Augumentation.py
```

Comment:
Add usage part also here

Reply:
Usage??? You mean test code?

Reply:
If you see in readme, there is a description of all functions used in files. Please add accordingly.

Reply:
I mean In order to test this out... I have to add a model, test data and all. So  it will be better if I doit in a test directory

Reply:
I am not saying you to write code in readme. Just write description about what your functions do in autoresults.py. 
please check readme present in flow2ml folder to get idea